src/app/features/stacks-transaction-request/stacks-transaction-signer.tsx

```diff
@@ -52,6 +51,7 @@ export function StacksTransactionSigner({
   onSignStacksTransaction,
   isMultisig,
 }: StacksTransactionSignerProps) {
+  const [isShowingHighFeeConfirmation, setIsShowingHighFeeConfirmation] = useState(false);
```

Comment:
I repeat this a bit but before we were storing this in global state via `jotai`. We could do with considering how it should work and improving how I've done it